hooks/cascading-scans/package.json

```diff
@@ -1,51 +1 @@
-{
-"name": "@securecodebox/hook-cascading-scans",
-  "version": "1.0.0",
-  "description": "secureCodeBox Hook to cascade scan in an declarative manner.",
-  "homepage": "https://www.secureCodeBox.io",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/secureCodeBox/secureCodeBox.git"
-  },
-  "main": "hook.js",
-  "scripts": {
-    "build": "npx typescript hook.ts --sourceMap",
-    "test": "jest . --verbose false"
-  },
-  "keywords": [
-    "secureCodeBox",
-    "security",
-    "hook"
-  ],
-  "author": {
-    "name": "iteratec GmbH",
-    "email": "securecodebox@iteratec.com",
-    "url": "https://www.iteratec.com"
-  },
-  "contributors": [
-    {
-      "name": "Jannik Hollenbach",
-      "url": "https://github.com/J12934"
-    },
-    {
-      "name": "Robert Seedorff",
-      "url": "https://github.com/rseedorff"
-    }
-  ],
-  "bugs": {
-    "url": "https://github.com/secureCodeBox/secureCodeBox/issues"
-  },
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@kubernetes/client-node": "^0.15.0",
-    "lodash": "^4.17.21",
-    "matcher": "^4.0.0",
-    "mustache": "^4.2.0"
-  },
-  "devDependencies": {
-    "@types/lodash": "^4.14.171",
-    "@types/node": "^16.3.3",
-    "jest": "^27.0.6",
-    "typescript": "^4.3.5"
-  }
-}
+{"name":"@securecodebox/hook-cascading-scans","version":"1.0.0","description":"secureCodeBox Hook to cascade scan in an declarative manner.","homepage":"https://www.secureCodeBox.io","repository":{"type":"git","url":"git+https://github.com/secureCodeBox/secureCodeBox.git"},"main":"hook.js","scripts":{"build":"npx typescript hook.ts --sourceMap","test":"jest . --verbose false"},"keywords":["secureCodeBox","security","hook"],"author":{"name":"iteratec GmbH","email":"securecodebox@iteratec.com","url":"https://www.iteratec.com"},"contributors":[{"name":"Jannik Hollenbach","url":"https://github.com/J12934"},{"name":"Robert Seedorff","url":"https://github.com/rseedorff"}],"bugs":{"url":"https://github.com/secureCodeBox/secureCodeBox/issues"},"license":"Apache-2.0","dependencies":{"@kubernetes/client-node":"^0.15.0","lodash":"^4.17.21","matcher":"^4.0.0","mustache":"^4.2.0"},"devDependencies":{"@types/lodash":"^4.14.171","@types/node":"^14.17.5","jest":"^27.0.6","typescript":"^4.3.5"}}
```

Comment:
I think it was not intended that all line breaks are gone in this file 😅

Reply:
I really do hate NPM